packages/zeego/src/menu/create-ios-menu/index.ios.tsx

```diff
@@ -231,9 +231,15 @@ If you want to use a custom component as your <Content />, you can use the menui
         ).targetChildren
 
         if (iconChildren?.[0]?.props.iosIconName) {
+          const iconConfiguration =
+            iconChildren?.[0]?.props.iosIconConfiguration
```

Comment:
Let's rename `iosIconConfiguration` → `ios`. And then in the if statement, check for `ios` too.

Then `systemName = iconConfiguration?.name ?? iconChildren[0].props.iosIconName`.

Reply:
Sounds good!